pkg/pillar/cmd/zedagent/attesttask.go

```diff
@@ -340,6 +342,10 @@ func (server *VerifierImpl) SendAttestEscrow(ctx *zattest.Context) error {
 	if !zedcloud.UseV2API() {
 		return zattest.ErrNoVerifier
 	}
+	if attestCtx.SkipEscrow {
+		log.Notice("[ATTEST] Escrow successful skipped")
+		return nil
```

Comment:
If we do this without an error will the attesttask state machine retry later when SkipEscrow is set to false?

This is an area we need to make sure is robust for a single boot of a device (that once the vault is created or unlocked that the keys are sent to the controller no matter what).

FWIW if/when this is robust we should also improve it if there is a power outage or crash immediately after the vault has been created on first boot (we've seen a case when the key was not sent due to a device being powered off at the wrong time, and on a subsequent boot it wasn't sent since there was no change - the TPM unsealed the key). But that can be a follow-on PR to make this escrow even more robust.

Reply:
I want to handle restart of attestation in case of 403 from config endpoint if no TPM enabled (we do not expect escrow key in that case but stay in waiting for escrow state).

Reply:
OK